crates/biome_html_formatter/tests/quick_test.rs

```diff
@@ -0,0 +1,44 @@
+use biome_formatter::{AttributePosition, IndentStyle, LineWidth};
+use biome_formatter_test::check_reformat::CheckReformat;
+use biome_html_formatter::context::HtmlFormatOptions;
+use biome_html_formatter::{format_node, HtmlFormatLanguage};
+use biome_html_parser::parse_html;
+use biome_html_syntax::HtmlFileSource;
+
+mod language {
+    include!("language.rs");
+}
+
+#[ignore]
+#[test]
+// use this test check if your snippet prints as you wish, without using a snapshot
+fn quick_test() {
+    let src = r#"
+<div>
+    <p>hello
+    </p>
+</div>
+
+    "#;
+    let source_type = HtmlFileSource::html();
+    let tree = parse_html(src);
+    let options = HtmlFormatOptions::new()
+        .with_indent_style(IndentStyle::Space)
+        .with_line_width(LineWidth::try_from(80).unwrap())
```

Comment:
consider using constants instead of magic numbers.

`const MAX_LINE_WIDTH: usize = 80;`

Reply:
This is code for the `quick_test`, and so I've tried to align it with the other `quick_test`s in other formatter crates. This makes the configuration easier to change when experimenting. IMO, extracting it into a const would be inconsistent with the other `quick_test`s, and slightly harm DX.